test/spirv-version-controls-negative-1.spt

```diff
@@ -0,0 +1,33 @@
+119734787 66560 393230 12 0·
```

Comment:
There's unusual punctuation at the ends of each of the lines in this test. Presumably this was from a file that rendered spaces at the ends of lines as "·". It would be good to remove this.

Reply:
Yes, you are right about rendering trailing whitespaces as "·" - will remove this

Reply:
Fixed